src/containers/AdminAutosending/components/AutosendingTargetRow.tsx

```diff
@@ -0,0 +1,124 @@
+import Button from "@material-ui/core/Button";
+import Chip from "@material-ui/core/Chip";
+import green from "@material-ui/core/colors/green";
+import grey from "@material-ui/core/colors/grey";
+import red from "@material-ui/core/colors/red";
+import { makeStyles } from "@material-ui/core/styles";
+import TableCell from "@material-ui/core/TableCell";
+import TableRow from "@material-ui/core/TableRow";
+import MoreIcon from "@material-ui/icons/ArrowForward";
+import PauseIcon from "@material-ui/icons/Pause";
+import PlayIcon from "@material-ui/icons/PlayArrow";
+import { AutosendingTargetFragment } from "@spoke/spoke-codegen";
```

Comment:
Nice to use a fragment as a component prop, I was struggling to get the right prop type for the row using accessors